wfl/pipeline/remote.py

```diff
@@ -75,15 +84,36 @@ def do_remotely(remote_info, hash_ignore=[], chunksize=1, iterable=None, configs
     # gather results and write them to original configset_out
     configset_out.pre_write()
     at_i = 0
-    for xpr in xprs:
+    for chunk_i, xpr in enumerate(xprs):
         if not quiet:
             sys.stderr.write(f'Gathering results for {xpr.id}\n')
-        ats_out, stdout, stderr = xpr.get_results(timeout=remote_info.timeout, check_interval=remote_info.check_interval)
-        for at in ats_out.group_iter():
-            configset_out.write(at, from_input_file=input_files[at_i])
-            at_i += 1
-        sys.stdout.write(stdout)
-        sys.stderr.write(stderr)
+
+        try:
+            ats_out, stdout, stderr = xpr.get_results(timeout=remote_info.timeout, check_interval=remote_info.check_interval)
+        except Exception as exc:
+            warnings.warn(f'Failed in remote job {xpr.id} on {xpr.system_name}')
+            if not remote_info.skip_failures:
+                raise
+            if len(all_items) > 0 and isinstance(all_items[chunk_i][0], Atoms):
+                # get ready to write input configs to output
+                ats_out = ConfigSet_in(input_configs=all_items[chunk_i])
```

Comment:
What about adding an `atoms.info["remote_job_errored"] = True` or something similar?

Reply:
Yes, not a bad idea.  I'll do that.

Reply:
Please resolve this conversation if you're OK with 79e6adc